docs/design/ovs-pipeline.md

```diff
@@ -671,7 +680,7 @@ use the SNAT IP. The flow matches the tunnel destination IP (which should be
 equal to the SNAT IP), and sets the 8 bits ID of the SNAT IP to pkt_mark.
 
 ```text
-table=71, priority=200,ct_state=+new+trk,ip,tun_dst="192.168.77.101" actions=load:0x1->NXM_NX_PKT_MARK[0..7],goto_table:80
+table=71, priority=200,ct_state=+new+trk,ip,tun_dst="192.168.10.101" actions=load:0x1->NXM_NX_PKT_MARK[0..7],goto_table:80
```

Comment:
Out of curiosity, Is the IP mapped by any example flow?

The example flow in L674 `load:0xc0a84d66->NXM_NX_TUN_IPV4_DST[]` maps to `192.168.77.102`

Reply:
Good point. Let me change 0xc0a84d66 to 0xc0a80a66